src/main/java/org/mqttbee/mqtt/mqtt3/Mqtt3ClientView.java

```diff
@@ -29,23 +28,57 @@
 import org.mqttbee.api.mqtt.mqtt3.message.publish.Mqtt3PublishResult;
 import org.mqttbee.api.mqtt.mqtt3.message.subscribe.Mqtt3Subscribe;
 import org.mqttbee.api.mqtt.mqtt3.message.unsubscribe.Mqtt3Unsubscribe;
+import org.mqttbee.api.mqtt.mqtt5.exceptions.Mqtt5MessageException;
+import org.mqttbee.api.mqtt.mqtt5.message.connect.connack.Mqtt5ConnAck;
+import org.mqttbee.api.mqtt.mqtt5.message.publish.Mqtt5Publish;
+import org.mqttbee.api.mqtt.mqtt5.message.publish.Mqtt5PublishResult;
 import org.mqttbee.mqtt.message.connect.connack.mqtt3.Mqtt3ConnAckView;
 import org.mqttbee.mqtt.message.connect.mqtt3.Mqtt3ConnectView;
 import org.mqttbee.mqtt.message.disconnect.mqtt3.Mqtt3DisconnectView;
 import org.mqttbee.mqtt.message.publish.mqtt3.Mqtt3PublishResultView;
 import org.mqttbee.mqtt.message.publish.mqtt3.Mqtt3PublishView;
 import org.mqttbee.mqtt.message.subscribe.mqtt3.Mqtt3SubscribeView;
 import org.mqttbee.mqtt.message.unsubscribe.mqtt3.Mqtt3UnsubscribeView;
+import org.mqttbee.mqtt.mqtt3.exceptions.Mqtt3ExceptionFactory;
 import org.mqttbee.mqtt5.Mqtt5ClientImpl;
 import org.mqttbee.util.MustNotBeImplementedUtil;
+import org.reactivestreams.Publisher;
+
 
 /**
  * @author Silvio Giebl
+ * @author David Katz
  */
 public class Mqtt3ClientView implements Mqtt3Client {
 
     private final Mqtt5ClientImpl wrapped;
 
+    private static final Function<? super Throwable, ? extends Single<? extends Mqtt5ConnAck>>
+        mapMqtt5toMqtt3ExceptionForConnect =
```

Comment:
naming convention for static fields: uppercase
=> MQTT5_TO_3_EXCEPTION_CONNACK_MAPPER